src/main/java/jackyy/dimensionaledibles/registry/ModConfig.java

```diff
@@ -203,6 +209,41 @@ public String toString() {
             public CustomCoords customCoords = new CustomCoords();
         }
 
+        public static class IslandCake implements CakeConfig {
+
+            @Config.Comment("Set the fuel used by Island Cake (Don't change this unless you know what you're doing).")
+            public String fuel = "minecraft:cobblestone";
+
+            @Config.Comment("Set to true to make the Island Cake pre-fueled upon placed.")
+            public boolean preFueled = false;
+
+            @Config.Comment("Set to true to make the Island Cake consume fuel.")
+            public boolean consumesFuel = true;
+
+            @Config.Comment("The Y-level that the Island Cake will teleport to")
+            @Config.RangeInt(min = 1, max = 256)
+            public int yLevel = 80;
+
+            @Config.Comment("The gap between two islands in regions")
+            @Config.RangeInt(min = 2, max = 100)
+            public int islandGapRegions = 2;
+
+            @Config.Comment("Allow players to create personal islands when on a FTB Team")
+            public boolean allowPersonalIslands = true;
+
+            @Config.Comment("The item that when used on an island cake turns it into a personal island cake")
+            public String personalLockingItem = "minecraft:diamond";
+
+            @Config.Comment("The dimension islands will be created in (dimension has to already exist)")
+            public Integer islandDimension = -2;
+
+            public String fuel(int dim) { return fuel; }
+            public boolean preFueled() { return preFueled; }
+            public boolean consumesFuel() { return consumesFuel; }
+            public boolean useCustomCoordinates(int dim) { return false; }
+            public CustomCoords customCoords(int dim) { return null; }
```

Comment:
Some spacing and formatting on these please.

Reply:
It's copied boilerplate code from the other cakes. They also have it.